test/torchaudio_unittest/common_utils/data_utils.py

```diff
@@ -75,7 +75,8 @@ def get_whitenoise(
     tensor.clamp_(-1.0, 1.0)
     if not channels_first:
         tensor = tensor.t()
-    return convert_tensor_encoding(tensor, dtype)
+    tensor = convert_tensor_encoding(tensor, dtype)
+    return tensor.to(device=device)
```

Comment:
seems like this was unintentionally removed in a previous refactoring job — added it back in

Reply:
nvm — it breaks other tests. will address elsewhere